service/README.md

```diff
@@ -0,0 +1,33 @@
+# Service lifecycle
```

Comment:
The main README uses title case, so let's stick with that (i.e., Service Lifecycle).